range_generator.py

```diff
@@ -0,0 +1,14 @@
+def MyRange(start, stop, step):
+    counter = start
+    while counter < (stop):
```

Comment:
Do we need brackets here?

Reply:
Brackets aren't supposed to be here, my omission